src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -155,42 +148,38 @@ private static void jsonSerializationExample(HazelcastInstance client) {
         selectCitiesByCountry(client, "AU");
```

Comment:
You use methods for each step, but the other example using the sql does not have the same structure. Can we make them use similar structures.

Reply:
OK, I splitted the SQL example to smaller chunks like this